corehq/apps/adm/models.py

```diff
@@ -584,22 +588,21 @@ class CaseFilterADMColumnMixin(DocumentSchema):
     case_types = ListProperty()
     case_status = StringProperty(default='', choices=[s[0] for s in CASE_STATUS_OPTIONS])
 
-    def get_filtered_cases(self, domain, user_id, include_groups=True, status=None, include_docs=False, datespan_keys=None):
-        if not datespan_keys:
-            datespan_keys = [[], [{}]]
+    def get_filtered_cases(self, domain, user_id, status=None,
+                           date_range=None):
```

Comment:
This function is only ever called (at least in code!) from `CaseCountADMColumn.raw_value`; nothing else subclasses `CaseFilterADMColumnMixin`.

A few things:
- can you confirm `CaseCountADMColumn` is still in use? If it's not, then easy—we can delete `case/by_date_modified_owner entirely`
- assuming it's still in use—it implements counting by doing full non-reduce queries, pulling all matching case object (just the metadata, but still wrapped) into a list in memory and then returning the length of the list. I can refactor this so that the helper functions ask couch for counts and then add them together, right?